phpcs.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards for Liveblog">
+
+	<rule ref="WordPress-VIP" />
+	<rule ref="WordPress-Extra" />
```

Comment:
There should be no spaces before closing `/>`. This is XML, not XHTML.

Reply:
Also copied from amp-wp.

Reply:
Fixed with 0ca4449.